### PR TITLE
Fixes #6553: Allow rudder server to remotely run the agent

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6553

Replacing previous PR: https://github.com/Normation/rudder-techniques/pull/663

### CODE REVIEW

File: initial-promises/node-server/common/1.0/cf-served.cf [hunk 1]
```diff
@@ -58,6 +58,11 @@ bundle server access_rules
         maproot => { host2ip("${server_info.cfserved}"), escape("${server_info.cfserved}")  },
         admit   => { host2ip("${server_info.cfserved}"), escape("${server_info.cfserved}")  };
 
+    any::
+      # Allow server to remotely run the agent
+      "/var/rudder/cfengine-community/bin/cf-agent"
+        admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}")) };
```
Comment (ncharles) [inline initial-promises/node-server/common/1.0/cf-served.cf]: string_downcase doesn't exist in 2.10 !
this code will fails

File: techniques/system/common/1.0/cf-served.st [hunk 2]
```diff
@@ -85,6 +85,10 @@ bundle server access_rules
         maproot => { host2ip("&POLICYSERVER&"), escape("&POLICYSERVER&")  },
         admit   => { host2ip("&POLICYSERVER&"), escape("&POLICYSERVER&")  };
 &endif&
+    any::
+      # Allow server to remotely run the agent
+      "/var/rudder/cfengine-community/bin/cf-agent"
+        admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}")) };
```
Comment (ncharles) [inline techniques/system/common/1.0/cf-served.st]: same here

**Comment by ncharles** [comment]
The code in this PR is invalid.
@peckpeck , you really need to test the code before pushing it to github :(

**Comment by peckpeck** [comment]
This was made for 3.1 and was subsequently backported

File: initial-promises/node-server/common/1.0/cf-served.cf [hunk 3]
```diff
@@ -82,7 +87,7 @@ bundle common def
     };
     !policy_server::
       "acl" slist => {
-      "${def.policy_server}"
+      "${def.policy_server}", host2ip("${def.policy_server}")
```
Comment (ncharles) [inline initial-promises/node-server/common/1.0/cf-served.cf]: this causes a massive slowdown of promise generation :(

**Pull request merged**